build.gradle

```diff
@@ -155,3 +167,44 @@ shadowJar {
     mergeServiceFiles()
     finalizedBy currentJar
 }
+
+// Generate ReadTools Online Doc files (jekyll formatted)
+task readtoolsDoc(type: Javadoc, dependsOn: ['cleanReadtoolsDoc', classes]) {
```

Comment:
This should not depend on `cleanReadtoolsDoc`, beacuse it removes the javadoc documentation if so.